applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/log.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+'require fs';
+'require poll';
+'require ui';
+'require view';
+
+return view.extend({
+	retrieveLog: async function() {
+		return Promise.all([
+			L.resolveDefault(fs.stat('/sbin/logread'), null),
+			L.resolveDefault(fs.stat('/usr/sbin/logread'), null)
+		]).then(function(stat) {
+			var logger = stat[0] ? stat[0].path : stat[1] ? stat[1].path : null;
+
+			return fs.exec_direct(logger, [ '-e', 'tailscale' ]).then(logdata => {
```

Comment:
This is nesting a promise inside a promise.
```suggestion
			const logdata = await fs.exec_direct(logger, [ '-e', 'tailscale' ]);
```